src/systemTest/java/com/flightstats/hub/kubernetes/HelmYamlOverride.java

```diff
@@ -0,0 +1,62 @@
+package com.flightstats.hub.kubernetes;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import com.flightstats.hub.system.config.HelmProperties;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.inject.Inject;
+import java.util.HashMap;
+import java.util.Map;
+
+@Slf4j
+public class HelmYamlOverride {
+    private static final ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
+    private final HelmProperties helmProperties;
+
+    @Inject
+    public HelmYamlOverride(HelmProperties helmProperties) {
+        this.helmProperties = helmProperties;
+    }
+
+    @SneakyThrows
+    String get() {
+        String yaml = mapper.writeValueAsString(build());
+        log.info("starting helm cluster install with yaml: {}", yaml);
+        return yaml;
+    }
+
+    private Map<String, Object> buildClusteredHub() {
+        Map<String, Object> clusteredHub = new HashMap<>();
+        clusteredHub.put(YamlKeys.CLUSTER_HUB_ENABLED, helmProperties.isHubInstallClustered());
+        return clusteredHub;
+    }
+
+    private Map<String, Object> buildHub() {
+        Map<String, Object> hub = new HashMap<>();
```

Comment:
Ever since @chriskessel showed off `Map.ofEntries` I've been wishing we were on Java 9. 😞

Reply:
feeling that